docs/pages/modules/third-party-library.mdx

```diff
@@ -94,6 +95,46 @@ dependencies {
     # Swift/Objective-C compatibility
 ```
 
+<Collapsible summary="Are you trying to use a .framework dependency?">
```

Comment:
```suggestion
<Collapsible summary={<>Are you trying to use a <CODE>.framework</CODE> dependency?</>}>
```

Let's use inline syntax highlight here. I've also added the import statement above.